Dockerfile

```diff
@@ -0,0 +1,138 @@
+# ARG base_image=ubuntu:18.04
+ARG base_image=python:3.8
+FROM $base_image
+
+USER root
+
+# Version strings and default paths
+ENV SPARK_VERSION=2.4.5 \
```

Comment:
Should those be ARGs instead? since they're used at docker build time ?

Or ARGs + ENV

Reply:
Yes - good call. I'll make that change.

Reply:
Done!